packages/cli/src/cli.ts

```diff
@@ -35,6 +35,7 @@ import type { Module } from '@wakaru/unpacker'
 enum Feature {
     Unpacker = 'Unpacker',
     Unminify = 'Unminify',
+    Perf = 'Perf',
```

Comment:
I think this one is not needed?

Reply:
Got it. I removed it and just pushed the updated code.